uniflow/op/model/model_server.py

```diff
@@ -474,3 +476,201 @@ def __call__(self, data: List[str]) -> List[str]:
+            response_body = json.loads(response.get("body").read())
+            return response_body.get("results")[0].get("outputText")
+
+        provider_output_preparation = {
+            "anthropic": prepare_anthropic_output,
+            "ai21": prepare_ai21_output,
+            "cohere": prepare_cohere_output,
+            "meta": prepare_meta_output,
+        }
+
+        prepare_output_for_provider = provider_output_preparation.get(provider, prepare_default_output)
+        return prepare_output_for_provider(response)
+    
+    def invoke_bedrock_model(
+        self,
+        prompt: str,
+        stop: Optional[List[str]] = None,
+        **kwargs: Any,
+    ) -> str:
+        """
+        Invokes the bedrock model with the given prompt and optional stop tokens.
+
+        Args:
+            prompt (str): The input prompt for the model.
+            stop (Optional[List[str]]): List of stop tokens to indicate the end of the generated text.
+            **kwargs: Additional keyword arguments to be passed to the model. Please refer to 
+            https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters.html for more details.
+
+        Returns:
+            str: The generated text from the bedrock model.
+
+        Raises:
+            ValueError: If there is an error raised by the bedrock service.
+        """
+        provider = self._get_provider()
+        _model_kwargs = self._model_config.model_kwargs or {}
+        params = {**_model_kwargs, **kwargs}
+
+        # Combine the prompt and model parameters into a single input body
+        input_body = self.prepare_input(provider, prompt, params)
+        body = json.dumps(input_body)
+        accept = "application/json"
+        contentType = "application/json"
+
+        # Invoke the model
+        try:
+            response = self._client.invoke_model(
+                body=body, modelId=self._model_config.model_name, accept=accept, contentType=contentType
+            )
+        except Exception as e:
+            raise ValueError(f"Error raised by bedrock service: {e}")
+
+        # Perform post-processing on the response
+        text = self.prepare_output(provider, response)
+
+        if stop is not None:
+            text = self.enforce_stop_tokens(text, stop)
+
+        return text
+
+    def __call__(self, data: List[str]) -> List[str]:
+        """Run model.
+
+        Current bedrock batch inference is implemented by creating asynchronous jobs. 
+        At present, we are not temporarily using Batch Inference. 
+        Reference: https://docs.aws.amazon.com/bedrock/latest/userguide/batch-inference-create.html
+
+        Args:
+            data List[str]: Data to run.
+
+        Returns:
+            str: Output data.
+        """
+        data = self._preprocess(data)
+        inference_data = []
+        for d in data:
+            inference_data.append(
+                self.invoke_bedrock_model(prompt = d, max_tokens_to_sample = 300)
```

Comment:
nit: same as another comment above. can we move `max_tokens_to_sample` into the config.

Reply:
I'll fix it right away.